spec/export/index.md

```diff
@@ -6,24 +6,24 @@ active: export
 
 ## Audience and Scope
 
-This implementation guide is intended to be used by developers of backend services (clients) and FHIR Resource Servers (e.g., EHR systems, data warehouses, and other clinical and administrative systems) that aim to interoperate by sharing large FHIR datasets.  The guide defines the application programming interfaces (APIs) through which an authenticated and authorized client may request a bulk-data export from a server, receive status information regarding progress in the generation of the requested files, and receive these files.  It also includes recommendations regarding the FHIR resources that might be exposed throuh the export interface.  
```

Comment:
@chris1uphealth  whats the difference here?

Reply:
The 4th bullet in that section had a typo, incuded was changed to included.

Reply:
I don't see any difference.